extensions/Worker.Extensions.Abstractions/src/ReferenceTypeAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Abstractions
+{
+    [AttributeUsage(AttributeTargets.Assembly)]
+    public class ReferenceTypeAttribute : Attribute
```

Comment:
Once we agree on the naming above, this attribute would follow suit. E.g. it could be called SupportsWorkerParameterBinding. And likely it should just be a marker attribute, not bool param needed?

Reply:
Ah this should be deleted.

We had discussed either introducing a new attribute (like this one) or using the existing extension information attribute to determine if an extension supports reference types. In the end we went with using the existing attribute so I should have removed this file.

In the `ExtensionInformationAttribute` file, we introduced a `SupportsBindingReferenceType` property - do you feel we should rename this and not use bool for that?